man/CMDLINE.template.md

```diff
@@ -0,0 +1,8 @@
+# Command line parameters
```

Comment:
Do we need both files, `man/CMDLINE.template` and `man/CMDLINE.template.md`?

Reply:
I'm thinking about adding CMDLINE.md to the wiki. But it seems it's not possible to automate it.

Reply:
So we can remove one of both?

Reply:
But now I am generating CMDLINE.md. I think we can manually copy its contents to the wiki every release.